.github/copilot-review-instructions.md

```
@@ -0,0 +1,27 @@
+﻿## Python Code Review Instructions
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)